modules/local/cooler/juicer.nf

```diff
@@ -0,0 +1,45 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from '../functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process JUICER {
+    tag "${meta.id}"
+    label 'process_medium'
+    label 'error_ignore'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    input:
```

Comment:
This process triggers an error when testing on AWS because it has no Docker container specified:

```
Error executing process > 'NFCORE_HICAR:HICAR:COOLER:JUICER (WT)'
Caused by:
  Invalid AWS Batch job definition -- provide a Docker image name or a Batch job definition name
Command executed:
  ## thanks https://www.biostars.org/p/360254/
  sort -k2,2d -k6,6d WT.10000.ginteractions > WT.10000.ginteractions.sorted
  resolutions=(500 1000 2000 5000 10000 20000 50000 100000 250000 500000 1000000 2000000 5000000)
  res=()
  function join_by { local IFS="$1"; shift; echo "$*"; }
  for i in ${resolutions[@]}
  do
      if [[ $i -ge 10000 ]]; then
          res+=($i)
      fi
  done
  res=$(join_by , ${res[@]})
  java -Xms512m -Xmx4096m -jar /.nextflow/assets/nf-core/hicar/bin/juicer_tools_1.22.01.jar pre         -r $res          --threads 2 WT.10000.ginteractions.sorted WT.10000.hic chr22.fa.sizes
 
  echo 1.22.01 > Juicer_tools.version.txt
```

Even if you're installing tools semi-manually within the pipeline, you need some kind of base container in the module. But this also ties into my review question about whether there is a better way to handle this tool.

Reply:
Now resolved: https://github.com/nf-core/hicar/blob/08910a81d26f8f5ff73d1565ae1eb2021ea7302a/modules/local/cooler/juicer.nf#L15-L20